benchmarks/transformer/score_mod.py

```diff
@@ -264,13 +264,29 @@ def relative_bias(score, b, h, m, n):
     def head_bias(score, b, h, m, n):
         return score + 2 * h
 
+    (bsz, n_heads, seq_len, head_dim) = shape
+    head_scale = torch.randn(n_heads, device="cuda", dtype=torch.float32)
+    batch_scale = torch.randn(bsz, device="cuda", dtype=torch.float32)
+    kv_scale = torch.randn(seq_len, device="cuda", dtype=torch.float32)
+    q_scale = torch.randn(seq_len, device="cuda", dtype=torch.float32)
+
+    def all_bias(score, batch, head, token_q, token_kv):
+        score = score + kv_scale[token_kv]
+        score = score + q_scale[token_q]
+        score = score + head_scale[head]
+        score = score + batch_scale[batch]
+        return score
+
```

Comment:
all_bias score mod added to evaluate the performance impact of modification masks.